src/ApiCheck/Program.cs

```diff
@@ -195,9 +211,20 @@ public static int Main(string[] args)
                 Console.WriteLine();
             }
 
-            if (differences.Count > 0)
+            var remainingExclusions = result.RemainingExclusions;
+            foreach (var exclusion in remainingExclusions)
+            {
+                Console.WriteLine($@"Exclusion not applied:
```

Comment:
BTW it would also be nice to either colorize or at least prefix each "error" case with something like "ERROR: " so that it's super clear it's a bad thing.